networking/aws_load_balancer_operator/installing-albo-sts-cluster.adoc

```diff
@@ -6,17 +6,41 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-You can install the AWS Load Balancer Operator on a Security Token Service (STS) cluster.
+You can install the AWS Load Balancer Operator (ALBO) on a Security Token Service (STS) cluster.
 
-The AWS Load Balancer Operator relies on `CredentialsRequest` to bootstrap the Operator and for each `AWSLoadBalancerController` instance. The AWS Load Balancer Operator waits until the required secrets are created and available. The Cloud Credential Operator does not provision the secrets automatically in the STS cluster. You must set the credentials secrets manually by using the `ccoctl` binary.
+The AWS Load Balancer Operator relies on a `CredentialsRequest` to bootstrap the Operator and for the `AWSLoadBalancerController` instance. The AWS Load Balancer Operator waits until the required secrets are created and available.
 
-If you do not want to provision credential secret by using the Cloud Credential Operator, you can configure the `AWSLoadBalancerController` instance on the STS cluster by specifying the credential secret in the AWS load Balancer Controller custom resource (CR).
+[id="creating-iam-role-albo-operator_{context}"]
+== Creating an IAM role for the AWS Load Balancer Operator
 
-include::modules/bootstrap-aws-load-balancer-operator.adoc[leveloffset=+1]
+An additional IAM role is required to successfully install the AWS Load Balancer Operator on a Security Token Service (STS) cluster. The IAM role is required to interact with subnets and Virtual Private Clouds (VPCs). The Operator generates a `CredentialsRequest` with this role to bootstrap itself.
 
-include::modules/configuring-albo-on-sts-cluster.adoc[leveloffset=+1]
+There are two options for creating the IAM role:
 
-include::modules/configuring-albo-on-sts-cluster-predefined-credentials.adoc[leveloffset=+1]
+* Using `ccoctl` and a predefined `CredentialsRequest`.
+* Using the AWS CLI and predefined AWS manifests.
+
+Use the AWS CLI if your environment does not support the `ccoctl` command.
+
+include::modules/using-ccoctl-create-iam-role-alb-operator.adoc[leveloffset=+2]
+include::modules/using-aws-cli-create-iam-role-alb-operator.adoc[leveloffset=+2]
+
+include::modules/specifying-role-arn-albo-sts.adoc[leveloffset=+1]
+
+[id="creating-iam-role-albo-controller_{context}"]
```

Comment:
Could all this be a concept module?

Reply:
I'm honestly not sure. At this point, I'm more interested in getting the PR merged than I am in reorganizing the content, but this is worth considering in the future.